src/components/Layout/Header/index.js

```diff
@@ -90,38 +99,35 @@ const Header = ({ isLogged, userImage, userName }) => {
         </div>
       </div>
       <div className="user-loging">
-        {isLogged ? (
+        {user ? (
           <>
             <Image
               preview={false}
-              src={userImage}
+              src={user.image}
               alt="user"
               className="userImage"
             />
-            <Text className="usernameTitle">{userName}</Text>
+            <Text className="usernameTitle">{user.name}</Text>
             <MainButton
               icon={<LogoutOutlined />}
               buttName="logout"
               id="logout"
               className="logout"
+              onClick={() => app.auth().signOut()}
```

Comment:
when the user log out 
the state user will be null
so you can make
setUser(null)